Assets/Scripts/Game/MagicAndEffects/EntityEffectManager.cs

```diff
@@ -464,20 +464,20 @@ public void AssignBundle(EntityEffectBundle sourceBundle, bool showNonPlayerFail
                     continue;
                 }
 
-                // Special handling for paralysis
-                if (effect is Paralyze)
+                // Saving throws
```

Comment:
Fixing this has exposed another bug. Player can now "save" against spells they cast on themselves (e.g. a simple healing spell). I'm sure I used to handle this, but it's been a while. If you don't mind, I'll send an edit to this PR to fix this one before merging.